[2022-02-09 15:29:45,045] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: connect_web3.wallet_task scheduled__2022-02-08T12:00:00+00:00 [queued]>
[2022-02-09 15:29:45,062] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: connect_web3.wallet_task scheduled__2022-02-08T12:00:00+00:00 [queued]>
[2022-02-09 15:29:45,062] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 15:29:45,063] {taskinstance.py:1239} INFO - Starting attempt 2 of 2
[2022-02-09 15:29:45,063] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 15:29:45,087] {taskinstance.py:1259} INFO - Executing <Task(WalletOperator): wallet_task> on 2022-02-08 12:00:00+00:00
[2022-02-09 15:29:45,092] {standard_task_runner.py:52} INFO - Started process 14132 to run task
[2022-02-09 15:29:45,104] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'connect_web3', 'wallet_task', 'scheduled__2022-02-08T12:00:00+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/connect_web3.py', '--cfg-path', '/var/folders/cm/mx_sf4z94vlfjw8ql31yvhgw0000gq/T/tmpaji7nz65', '--error-file', '/var/folders/cm/mx_sf4z94vlfjw8ql31yvhgw0000gq/T/tmpge1q_ii5']
[2022-02-09 15:29:45,114] {standard_task_runner.py:77} INFO - Job 8: Subtask wallet_task
[2022-02-09 15:29:50,208] {logging_mixin.py:109} INFO - Running <TaskInstance: connect_web3.wallet_task scheduled__2022-02-08T12:00:00+00:00 [running]> on host Veronicas-MacBook-Air.local
[2022-02-09 15:29:55,253] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=connect_web3
AIRFLOW_CTX_TASK_ID=wallet_task
AIRFLOW_CTX_EXECUTION_DATE=2022-02-08T12:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-08T12:00:00+00:00
[2022-02-09 15:29:55,323] {wallet_operator.py:28} INFO - Are we connected to w3? -> True
[2022-02-09 15:29:55,323] {wallet_operator.py:29} INFO - Balance of wallet: 100000000000000000000
[2022-02-09 15:29:55,343] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=connect_web3, task_id=wallet_task, execution_date=20220208T120000, start_date=20220209T202945, end_date=20220209T202955
[2022-02-09 15:29:55,378] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 15:29:55,401] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
